packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -52,3 +52,27 @@ export async function processProviderRequests(req: Request, res: Response) {
   const body = { ok: true, data: providers.scrub(updatedState) };
   res.status(200).send(body);
 }
+
+export async function testApi(req: Request, res: Response) {
+  const apiKey = req.header('x-api-key');
+  if (!apiKey || apiKey !== config.getEnvValue('HTTP_GATEWAY_API_KEY')) {
+    res.status(401).send({ error: 'Wrong API key' });
+  }
+
+  const parameters = req.body.parameters;
+  const endpointId = req.query.endpointId;
+
+  if (!endpointId) {
+    res.status(400).send({ error: 'Missing endpointId' });
+    return;
+  }
+
+  const [err, result] = await handlers.testApi(parsedConfig, endpointId as string, parameters);
+  if (err) {
+    res.status(400).send({ error: err.toString() });
+    return;
+  }
+
+  // NOTE: We do not want the user to see {"value": <actual_value>}, but the actual value itself
```

Comment:
Why is this?

Reply:
That was a change introduced by @Siegrift to an AWS HTTP gateway, I just wanted to keep it consistent. I think he'll be able to answer.

Reply:
It's as the comment say. If you did a request in http gw you would have a response:
```
{"value": {"rawValue": ..., "encodedValue": ..., "values": [...]}}
```

that is unnecessary an you isntead want the reponse flattened, like described in docs https://docs.api3.org/airnode/v0.4/grp-providers/guides/build-an-airnode/http-gateway.html#using-curl

Reply:
It could be refactored so that this wrapping is removed and the flattened value is only returned.